.github/workflows/cicd.yml

```
@@ -978,17 +1132,24 @@ jobs:
       - name: Compute RC version
         id: version
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:33: Double quote to prevent globbing and word splitting

(shellcheck)